python/lsst/pipe/tasks/functors.py

```diff
@@ -1004,3 +1004,184 @@ def columns(self):
+
+    def instFluxToMagnitude(self, instFlux, localCalib):
+        """Convert instrument flux to nanojanskys.
+
+        Parameters
+        ----------
+        instFlux : `numpy.ndarray` or `pd.Series`
+            Array of instrument flux measurements
+        localCalib : `numpy.ndarray` or `pd.Series`
+            Array of local photometric calibration estimates.
+
+        Returns
+        -------
+        calibMag : `numpy.ndarray` or `pd.Series`
+            Array of calibrated AB magnitudes.
+        """
+        return -2.5 * np.log10(self.instFluxToNanojansky(instFlux, localCalib)) + self.logNJanskyToAB
+
+    def instFluxErrToMagnitudeErr(self, instFlux, instFluxErr, localCalib, localCalibErr):
+        """Convert instrument flux to nanojanskys.
+
+        Parameters
+        ----------
+        instFlux : `numpy.ndarray` or `pd.Series`
+            Array of instrument flux measurements
+        instFluxErr : `numpy.ndarray` or `pd.Series`
+            Errors on associated ``instFlux`` values
+        localCalib : `numpy.ndarray` or `pd.Series`
+            Array of local photometric calibration estimates.
+        localCalibErr : `numpy.ndarray` or `pd.Series`
+           Errors on associated ``localCalib`` values
+
+        Returns
+        -------
+        calibMagErr: `numpy.ndarray` or `pd.Series`
+            Error on calibrated AB magnitudes.
+        """
+        err = self.instFluxErrToNanojanskyErr(instFlux, instFluxErr, localCalib, localCalibErr)
+        return 2.5 / np.log(10) * err / self.instFluxToNanojansky(instFlux, instFluxErr)
+
+
+class LocalNanojansky(LocalPhotometry):
+    """Compute calibrated fluxes using the local calibration value.
+    """
+
+    @property
+    def columns(self):
+        return [self.instFluxCol, self.photoCalibCol]
+
+    @property
+    def name(self):
+        return 'flux_{0}'.format(self.instFluxCol)
+
+    def _func(self, df):
+        return self.instFluxToNanojansky(df[self.instFluxCol], df[self.photoCalibCol])
+
+
+class LocalNanojanskyErr(LocalPhotometry):
+    """Compute calibrated flux errors using the local calibration value.
+    """
+
+    @property
+    def columns(self):
+        return [self.instFluxCol, self.instFluxErrCol,
+                self.photoCalibCol, self.photoCalibErrCol]
+
+    @property
+    def name(self):
+        return 'fluxErr_{0}'.format(self.instFluxCol)
+
+    def _func(self, df):
+        return self.instFluxErrToNanojanskyErr(df[self.instFluxCol], df[self.instFluxErrCol],
+                                               df[self.photoCalibCol], df[self.photoCalibErrCol])
+
+
+class LocalMagnitude(LocalPhotometry):
+    """Compute calibrated AB magnitudes using the local calibration value.
+    """
+
+    @property
+    def columns(self):
+        return [self.instFluxCol, self.photoCalibCol]
+
+    def _func(self, df):
+        return self.instFluxToMagnitude(df[self.col], self.fluxMag0)
+
+
+class LocalMagnitudeErr(Photometry):
```

Comment:
Should this be `LocalPhotometry`?

Reply:
Don't know why this showed up in your diff? This was something I found and fixed during building the unittest. Github here even says it's outdated which is weird.